### PR TITLE
[IMP] accounting: update the bank accounts documentation

### PR DESCRIPTION
**Pull request by Donapi**
[task 2607774](https://www.odoo.com/web#id=2607774&cids=1&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 1]
```diff
@@ -1,95 +1,81 @@
...
+page opens. Click on the external sign next to the :guilabel:`Bank` field and update your bank
+information accordingly. These details are used when registering payments.
 
-You can configure which types of payments are enabled in the **Payment Method Types** section and
-how the bank statements are recorded and posted in the **Bank Statements** section.
-
-.. image:: bank_accounts/bank_accounts_configuration.png
-   :align: center
-   :alt: Advanced configuration of a bank account in Odoo Accounting
+The :guilabel:`Bank Feeds` can be selected from the bank journal. 3 options are possibles:
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:75]:
```suggestion
:guilabel:`Bank Feeds` can be selected from the bank journal. Three options are available:
```
Numbers from zero to nine are usually spelled out. Always spell out numbers when they start a sentence.

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 2]
```diff
@@ -1,95 +1,81 @@
...
-.. seealso::
-   * :doc:`../../bank/feeds/bank_synchronization`
-   * :doc:`../feeds/bank_statements`
-   * `Odoo Tutorials: Accounting Basics <https://www.odoo.com/r/lsZ>`_
\ No newline at end of file
+- :guilabel:`Undefined yet`, should be selected if you don’t know yet if you will sync your bank
+  with Odoo or not.
+- :guilabel:`Import (CAMT,CODA, CSV, OFX, QIF)`, should be selected if you want to import your bank
+  statement via a different format.
+- :guilabel:`Automated Bank Synchronization`
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:81]:
```suggestion
- :guilabel:`Undefined yet`, which should be selected when you don’t know yet if you will sync your bank account
  with your database or not.
- :guilabel:`Import (CAMT, CODA, CSV, OFX, QIF)`, which should be used if you want to import your bank
  statements using a different format.
- :guilabel:`Automated Bank Synchronization`
```
It feels weird that you don't explain the last point, even if obvious.

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 3]
```diff
@@ -1,95 +1,81 @@
...
 
-Every **Bank Journal** is displayed by default on the **Accounting Overview** in the form of a
-convenient card. It includes action buttons that are displayed when appropriate.
+The :guilabel:`Bank Journals` are displayed by default on the :guilabel:`Accounting Dashboard` in
+the form of convenient cards that and include action buttons.
 
 .. image:: bank_accounts/bank_accounts_card.png
    :align: center
-   :alt: Bank Journals Cards are displayed on the Accounting Overview in Odoo Accounting
+   :alt: Bank journals are displayed on the Accounting Dashboard and contain actions buttons
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/bank/setup/bank_accounts.rst:21]: here we don't mean parameters, it's really the actions, ie: 1 bill to pay

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Don't forget to `make html` - that's why you failed the documentation check (if you click on it, you will see the error _No "build succeeded." found in logs._)

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 4]
```diff
@@ -1,95 +1,81 @@
...
-To edit an existing bank account, go to :menuselection:`Accounting --> Configuration --> Bank
-Accounts`, and open the bank account you want to modify.
+To **edit an existing bank account** go to :menuselection:`Accounting --> Configuration --> Journals`
+and select the bank journal you want to modify.
 
-If you need to edit the bank account details, go to the *Bank Account* field and click on the
-*External Link* button next to the list arrow. There, you can edit the bank account's number,
-Account Holder, Account Holder Name, and your Bank Institution's details by clicking on the
-*External Link* next to the *Bank* field. These details are used to register some payments.
+If you need to **edit your bank account details**, click on the :guilabel:`Account number`. A new
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/bank/setup/bank_accounts.rst:71]: no you really have to click on the bank account 
image
image

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 5]
```diff
@@ -1,95 +1,81 @@
...
-.. image:: bank_accounts/bank_accounts_connect.png
-   :align: center
-   :alt: Select a bank institution in the list and connect it to Odoo Accounting
-
 Manual configuration
 --------------------
 
-If your Bank Institution can’t be synchronized automatically, or if you prefer not to sync it with
+If your bank institution can’t be synchronized automatically, or if you prefer not to sync it with
 your database, you may also configure your bank account manually.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/bank/setup/bank_accounts.rst:47]: I updated as per most of your comments

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 6]
```diff
@@ -1,95 +1,81 @@
...
 
-- **Name**: the bank account's name, as displayed on Odoo.
-- **Account Number**: your bank account number (IBAN in Europe).
-- **Bank**: click on *Create and Edit* to configure the bank's details. Add the bank institution's
-  name and its Identifier Code (BIC or SWIFT).
-- **Code**: this code is your Journal's *Short Code*, as displayed on Odoo. By default, Odoo creates
-  a new Journal with this Short Code.
-- **Journal**: This field is displayed if you have an existing Bank Journal that is not linked yet
-  to a bank account. If so, then select the *Journal* you want to use to record the financial
-  transactions linked to this bank account or create a new one by clicking on *Create and Edit*.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/bank/setup/bank_accounts.rst:64]: Not sure it is really needed to describe those fields, and a short description of the fields is available when moving the computer mouse over it.

**Comment by Donapi** [comment]
@xpl-odoo @toaa-odoo Thanks for your review guys!

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 7]
```diff
@@ -1,95 +1,85 @@
...
 
-.. seealso::
-   * :doc:`../../bank/feeds/bank_synchronization`
-   * :doc:`../feeds/bank_statements`
-   * `Odoo Tutorials: Accounting Basics <https://www.odoo.com/r/lsZ>`_
\ No newline at end of file
+:guilabel:`Bank Feeds` can be selected from the bank journal to define how your bank statements are
+registered. Three options are available:
+
+- :guilabel:`Undefined yet`, which should be selected when you don’t know yet if you will synchronize your
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:80]: Line break after 100 characters

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 8]
```diff
@@ -1,95 +1,85 @@
...
-instructions on-screen.
+To connect your bank account to your database, go to :menuselection:`Accounting --> Configuration
+--> Banks: Add a Bank Account`. Select your bank in the list, click on :guilabel:`Connect`, and
+follow the instructions.
 
-.. note::
-   :doc:`Click here <../../bank/feeds/bank_synchronization>` for more information about this bank
-   synchronization.
+.. seealso::
+   :doc:`../../bank/feeds/bank_synchronization`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:38]:
```suggestion
   :doc:`../../bank/feeds/bank_synchronization`
```
Not a sentence, full stop isn't needed

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 9]
```diff
@@ -1,95 +1,85 @@
...
+---------------------
 
-Manual configuration
---------------------
+If your banking institution is not available in Odoo, or if you don't want to connect your bank
+account to your database, you can configure your bank account manually.
 
-If your Bank Institution can’t be synchronized automatically, or if you prefer not to sync it with
-your database, you may also configure your bank account manually.
+A default bank journal is available and can be used to configure your bank account, by going to
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:46]:
```suggestion
A default bank journal is available and can be used to configure your bank account by going to
```
No comma

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 10]
```diff
@@ -1,95 +1,85 @@
...
+In Odoo Accounting, each bank account has a dedicated journal set to post all entries in a dedicated
+account. Both the journal and the account are automatically created and configured whenever you add
+a bank account.
 
-.. note:: Whenever you add a Bank Account, a dedicated journal and a dedicated account are
-   automatically created and configured.
-
-Every **Bank Journal** is displayed by default on the **Accounting Overview** in the form of a
-convenient card. It includes action buttons that are displayed when appropriate.
+:guilabel:`Bank journals` are displayed by default on the :guilabel:`Accounting Dashboard` in
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/setup/bank_accounts.rst:13]:
```suggestion
:guilabel:`Bank journals` are displayed by default on the :guilabel:`Accounting Dashboard` in
```
Do `Bank journals` appear on the UI? (just asking, if not, no guilabel ;))

**Review by xpl-odoo — APPROVED** [review]
Very small comments/suggestions, but it's good to go otherwise :)

**Review by toaa-odoo — APPROVED** [review]

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @Donapi @toaa-odoo and @xpl-odoo 

At last, I review it :D 
Thanks for the improvements.

I think you can improve the structure, though.

Currently, you have first a heading that says "create a new bank account" where the first instruction is actually to edit the existing one. Next heading (different level) explains how to "edit an existing account". And both headings contain content that is valid both for creation and editing.

I suggest you have a single heading for all that content, and sub-headings for all the different topics: bank account number, bank feeds, ... but also the suspense account, profit/loss accounts, currency. That's the kind of info that is useful in the documentation
+ links to related pages (currencies, outstanding accounts etc) 
Speaking of outstanding accounts, other tabs are still part of the bank configuration. But you can have that in separate docs if you prefer.

technical

- there are two commits. It looks like you intended to have only one. You squash them together (`git rebase -i 15.0` and `squash` the second commit instead of `pick`)
- The PR name isn't correct (there is a way to squash commits when we merge, and the resulting commit takes the name of the PR)
- filenames don't follow the [media files guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files)
  - hyphens `-` instead of underscores `_`
  - no need to rewrite the prefix bank-accounts in the filename since it's already in the folder name
-

**Review by jcs-odoo — APPROVED** [review]
Hi @Donapi 

I made a few changes and additions to your commit.
Mostly to move your content around without changing it, but also to add some links, a note in the intro, and a short section on how to add a cash journal.

[You can review my changes here](https://github.com/odoo/documentation/compare/9bf4ff8081ec9c2f766c85b4dcb366aafefad54f..98676d937fa6afbcd1560af43b07b19ce51479d1), but it looks heavier than it really is, since moving some paragraphs to another place looks like changing a lot of content.

Have a good day and thanks for the PR :)

File: content/applications/finance/accounting/bank/setup/bank_accounts.rst [hunk 11]
```diff
@@ -1,95 +1,136 @@
...
-You can either connect your bank account to your Odoo database, or configure your bank account
-manually and :doc:`upload the bank statements manually <../feeds/bank_statements>`.
+To connect your bank account to your database, go to :menuselection:`Accounting --> Configuration
+--> Banks: Add a Bank Account`, select your bank in the list, click on :guilabel:`Connect`, and
+follow the instructions.
 
-Bank Synchronization
---------------------
+.. seealso::
+   - :doc:`../../bank/feeds/bank_synchronization`
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/bank/setup/bank_accounts.rst]: Agreed!

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**